scripts/helpers/tizen_gdbserver_run.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+SDB="$TIZEN_SDK_ROOT/tools/sdb"
```

Comment:
You've added `$TIZEN_SDK_ROOT/tools` to the PATH, so I think that simple `sdb` will be good enough.

Reply:
Fixed